docs/detections/alerts-view-details.asciidoc

```diff
@@ -26,7 +26,11 @@ The *Overview* tab contains these features:
 
 * *Summary*: Displays general details such as the alert's status, severity, risk score, and a link to the detection rule that produced the alert.
 
-* *Reason statement*: Provides a description of what generated the alert and provides general alert details. You can use this to understand the alert's origin and determine if the alert is relevant to your investigation.
+* *Alert-rendered reason statement*: Summarizes alert details and assembles the information into a human-readable format. High-level details are provided, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Alert fields are interactive. Hover over alert fields to access multiple actions.
```

Comment:
Note to self: Need to refresh `alert-details-flyout.png` to show the updated reason statement and event renderer. Screenshot should also include the updated versions of the insights sections.

Reply:
Done